content/docs/gke/authentication.md

```diff
@@ -183,6 +183,79 @@ By default, `Roles` and `RoleBindings` apply only to resources in a specific nam
 You can find more information in the 
 [Kubernetes docs](https://kubernetes.io/docs/reference/access-authn-authz/rbac/#role-and-clusterrole).
 
+### Namespaces
```

Comment:
I think this belongs at the top of the page
https://deploy-preview-957--competent-brattain-de2d6d.netlify.com/docs/gke/authentication/

The first section is "in-cluster-authentication" and I think this text belongs there because it involves credentials from within a namespace.

The existing opening paragraph

> When you set up Kubeflow for GCP, it will automatically provision three service accounts with different privileges. In particular, the ${KFAPP}-user service account is meant to grant your user services access to GCP. The credentials to this service account can be accessed within the cluster as a Kubernetes secret.

Should probably be revised/replaced/merge with the paragraph you have here "starting from version 0.6".

Reply:
I moved it to the top but I'm not clear as to why it would be revised with that opening paragraph since it seems like the information stands on its own?